bin/heroku-web

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
```

Comment:
```suggestion
#!/usr/bin/env bash
```

and you are missing a new line.

Reply:
Should we added a `set -e` below this line?

Reply:
`set -eou pipefail` if not commands fail.